lib/simplifyExpression/basicsSearch/convertMixedNumberToImproperFraction.js

```diff
@@ -77,21 +79,23 @@ function convertToUnsimplifiedImproperFraction(
   }
 
   return Node.Status.nodeChanged(
-    ChangeTypes.IMPROPER_FRACTION_NUMERATOR, node, newNode, false);
+    ChangeTypes.IMPROPER_FRACTION_NUMERATOR, oldNode, newNode, false);
 }
 
 // Simplify multiplication in the numerator of an improper fraction
 // e.g. ((1 * 3) + 2) / 3 -> (3 + 2) / 3
 function simplifyMultiplicationInImproperFraction(
-  node, wholeNumber, numerator, denominator, isNegativeMixedNumber) {
+  oldNode, wholeNumber, numerator, denominator, isNegativeMixedNumber) {
 
-  let newNode = clone(node);
+  let newNode = clone(oldNode);
```

Comment:
wait do you even use newNode? you set it below on line 104, just declare it there :)